src/WebJobs.Extensions.DurableTask/AzureStorageDurabilityProviderFactory.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
 {
     internal class AzureStorageDurabilityProviderFactory : IDurabilityProviderFactory
     {
+        private const string LoggerName = "AzureStorage";
```

Comment:
Should we use a logger name that is consistent with the existing Azure Functions logger names? I worry that "AzureStorage" will be confusing for users since it doesn't have "Functions" anywhere in the name. Ideally it's consistent with our other log category names. How about "Host.Triggers.DurableTask.AzureStorage" so that it's effectively a subsection of the existing "Host.Triggers.DurableTask" category?